devnet/cross-client/docker-compose.yml

```diff
@@ -0,0 +1,47 @@
+services: 
+  alice:
+    platform: linux/amd64
+    build:
+      context: ./../..
+      dockerfile: devnet/alice.Dockerfile
+      args:
+        DD_API_KEY: ${DD_API_KEY}
+    ports:
+      - 7001
+      - 8545
+      - 8546
+      - 8540
+      - 9876
+      - 6060
```

Comment:
do we expose pprof on bob?

Reply:
yes, bind pprof port (`6060`) to bob gossamer node.